ledger/participant-integration-api/src/main/resources/db/migration/postgres/V128__etq_tables_and_indexes.sql

```diff
@@ -0,0 +1,62 @@
+
+-- Flat transactions
+
+ALTER TABLE participant_events_create_filter RENAME
```

Comment:
Probably there's a good reason behind it, but not clear to me. Why are we renaming this guy?

Reply:
This is in order to match the naming of the new filter tables:

```  
pe_create_id_filter_stakeholder  -- previously participant_events_create_filter
pe_create_id_filter_non_stakeholder_informee
pe_consuming_id_filter_stakeholder
pe_consuming_id_filter_non_stakeholder_informee
pe_non_consuming_id_filter_informee
```